examples/guides/se-restful-webservice/src/main/java/io/helidon/guides/se/restfulwebservice/Main.java

```diff
@@ -19,30 +19,32 @@
 import java.io.IOException;
 import java.util.logging.LogManager;
 
-// tag::importsHealth1[]
+import javax.json.Json;
+import javax.json.JsonObject;
+
 import io.helidon.common.http.Http;
+import io.helidon.config.Config;
+import io.helidon.metrics.MetricsSupport;
+import io.helidon.webserver.Routing;
+import io.helidon.webserver.ServerConfiguration;
+import io.helidon.webserver.ServerRequest;
+import io.helidon.webserver.ServerResponse;
+import io.helidon.webserver.WebServer;
+import io.helidon.webserver.json.JsonSupport;
+
+// tag::importsHealth1[]
 // end::importsHealth1[]
 // tag::importsStart[]
-import io.helidon.config.Config;
```

Comment:
These tags are included in guides/01_SE_REST_web-service.adoc and your changes remove their content.
The pipeline failures seems to be that the sitegen template `vuetify/block_listing` does expect a non empty value and then freemarker throws an exception. The template should be improved to handle this better: either fail nicely or allow an empty value. I'll file a bug on the build tools repo,

Reply:
filed https://github.com/oracle/helidon-build-tools/issues/14

Reply:
My mistake - automatic reordering of imports in IDE caused this. I have disabled it, reverted the change and now it should pass.